editoast/editoast_schemas/src/infra/switch_type.rs

```diff
@@ -24,6 +24,7 @@ editoast_common::schemas! {
 #[derive(Debug, Derivative, Clone, Deserialize, Serialize, PartialEq, Eq, ToSchema)]
 #[serde(deny_unknown_fields)]
 #[derivative(Default)]
+#[schema(example = json!({"id": "Point", "ports": ["LEFT", "RIGHT", "BASE"], "groups": { "LEFT": { "src": "BASE", "dst": "LEFT" }, "RIGHT": { "src": "BASE", "dst": "RIGHT" }}}))]
```

Comment:
minor: Do you think it's possible to help a little bit the formatting here? I believe `rustfmt` doesn't format code inside macro's call (`json!()` here). But maybe we can do it a little bit manually?

Reply:
Done.